pages/docsprint/guides/participation.md

```diff
@@ -4,4 +4,62 @@ permalink: /docsprint/guides/participation/
 
 # Doc Sprint Participation Guide
 
-Here will be a guide to participating in a doc sprint.
+A well-organized doc sprint is designed to improve team documentation quickly and efficiently. Making sure 18F's documentation is accessible and comprehensive is critical for scaling what we do.  
+
+Documentation includes, but is not limited to:
+
+* READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
+* Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
+* Updating our [18F Guides](http://18f.github.io/guides/)
```

Comment:
Should we specify that we're looking for design / content help on these guides? Where would that information go?